src/components/VerticalNavigation/VerticalNavigationTertiaryItem.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { getContext } from 'recompose';
+import { wrongDepth, correctDepth, navContextTypes } from './constants';
+import VerticalNavigationItemHelper from './VerticalNavigationItemHelper';
+
+/**
+ * VerticalNavigation.TertiaryItem - a child component for tertiary nav items under VerticalNavigation
+ * This is a thin wrapper for VerticalNavigationItemHelper, to expose the warnings and the explicit-depth API.
+ * In the future, we could lift some of the tertiary-specific code from Helper into this file.
+ */
+const BaseVerticalNavigationTertiaryItem = props => {
+  if (wrongDepth(props, 'tertiary')) return correctDepth(props);
+  return <VerticalNavigationItemHelper {...props} />;
+};
+
+BaseVerticalNavigationTertiaryItem.propTypes = {
+  ...VerticalNavigationItemHelper.propTypes
+};
+
+const VerticalNavigationTertiaryItem = getContext(navContextTypes)(
+  BaseVerticalNavigationTertiaryItem
+);
+
+VerticalNavigationTertiaryItem.propTypes =
```

Comment:
I seem to see the `propTypes` extensions being sometimes referenced directly vs using the `...` destructuring assignment. Can we just be consistent and use the destructuring assignment everywhere... ie:

```
VerticalNavigationTertiaryItem.propTypes = { ...BaseVerticalNavigationTertiaryItem.propTypes;
}
```

Reply:
Yeah, agreed. I meant to use the destructuring everywhere, if I missed it anywhere else I'll fix it.